mitmproxy/command.py

```diff
@@ -37,7 +37,7 @@ def typename(t: type) -> str:
     """
     to = mitmproxy.types.CommandTypes.get(t, None)
     if not to:
-        raise NotImplementedError(t)
+        raise SyntaxError("missing type annotations in consoleaddons.py")
```

Comment:
Two things here. That consoleaddons.py in this error string is obviously a leftover from debugging or something. :) This should be the command path, and you may have to modify the intermediary functions to make sure you have access to it. Secondly, `SyntaxError` is not the right exception here - it's a builtin that has a very specific meaning. Have a look at `mitmproxy.exceptions` for an appropriate exception here - perhaps `mitmproxy.exceptions.TypeError`?

I haven't validated the approach here deeply yet. Once these things are fixed, I'll take a closer look.